src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier(random);
+        Lottery lottery = new Lottery(random, colorSupplier);
+        for (int i = 0; i < 3; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.println(ball);
+        }
+
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/185403390-ce055569-9648-4026-b914-eb17337454aa.png)

Reply:
Pls review my jv-oop, i missclick in close pull request.